test/remote/gateways/remote_quickpay_v10_test.rb

```diff
@@ -128,15 +140,26 @@ def test_successful_purchase_and_credit
     assert_success credit
   end
 
+  def test_unsuccessful_authorization_capture_and_credit
+    assert auth = @gateway.authorize(@amount, @refund_rejected_card, @options)
+    assert_success auth
+    assert !auth.authorization.blank?
+    assert capture = @gateway.capture(@amount, auth.authorization)
+    assert_success capture
+    assert refund = @gateway.refund(@amount, auth.authorization)
+    assert_failure refund
+    assert_equal 'Rejected test operation', refund.message
+  end
+  
   def test_successful_store
-    assert response = @gateway.store(@valid_card, @options.merge(:description => 'test'))
-    assert_success response
+    assert response = @gateway.store(@valid_card, @options.merge(:description => 'test', :currency => 'USD', :amount => @amount))
+    assert_success response  
```

Comment:
Another minor thing, have some extra whitespace here as well as lines 161 and 162.